### ISSUE TITLE
Docker matrix config using wrong parameters

### ISSUE THREAD

**Issue by rabidllama** [GIScience/openrouteservice#721]
The `app.config.sample` in the Docker setup is set to use incorrect parameters in the Matrix area. Currently it is using
```
"matrix":{
            "enabled":true,
            "maximum_locations":200,
            "maximum_visited_nodes":100000,
            "allow_resolve_locations":true,
            "attribution":"openrouteservice.org, OpenStreetMap contributors"
         },
```
whereas the `app.config.sample` for the "normal" deploy uses
```
"matrix": {
        "enabled": true,
        "maximum_routes": 100,
        "maximum_routes_flexible": 25,
        "maximum_search_radius": 5000,
        "maximum_visited_nodes": 100000,
        "allow_resolve_locations": true,
        "attribution": "openrouteservice.org, OpenStreetMap contributors"
      },
```

The Docker config needs updating to reflect the correct properties which are in the normal config sample

**Comment by nilsnolde** [comment]
fixed in #717 

people always forget that there's another `app.config` when they change the source `app.config` (and can't blame them). smth we should get rid of, but for now updating the PR template to specifically state to update both files would be helpful.

**Comment by nilsnolde** [comment]
Just remember why I didn't attempt to get rid of docker's app.config: the paths for OSM file, elevation etc, which point to container paths. Can still be done of course with `jq` or so in the Dockerfile to replace those paths.. A separate issue though. But it's right, it'd be worth it.
